pendulum/date.py

```diff
@@ -137,11 +137,11 @@ def day_of_year(self):
 
         :rtype: int
         """
-        k = 1 if self.is_leap_year else 2
+        k = 1 if self.is_leap_year() else 2
 
         return (
             (275 * self.month) // 9
-            - k * (self.month + 9) // 12
+            - k * ((self.month + 9) // 12)
```

Comment:
Actually you are right: the parentheses are necessary since, otherwise, the result will be off by 1 day for non leap years.

Could you add a test covering this?